helpdesk_mgmt_timesheet_readonly/security/helpdesk_mgmt_timesheet_readonly.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data noupdate="0">
+
+        <record id="group_helpdesk_timesheet_readonly" model="res.groups">
+            <field name="name">User: Readonly tickets &amp; timesheets</field>
+            <field name="category_id" ref="module_helpdesk_timesheet_category"/>
+        </record>
+
+    </data>
+
+    <data noupdate="1">
+
+        <record id="helpdesk_ticket_timesheet_user_rule" model="ir.rule">
```

Comment:
I assume that the perm_read will be by default True, but would it not be clearer if it was indicated?

Reply:
Conventions say that by default `perm_*` fields are `True` , thus there's no need to fill them. But, in this case I agree with you, the clearest way is filling all the fields. I'll change it